pkg/server/app/fake_test.go

```diff
@@ -162,6 +162,31 @@ func TestFakeOperationsInfoErrNotFound(t *testing.T) {
 	}
 }
 
+func TestFakeOperationsList(t *testing.T) {
+	fake := NewFakeOperations()
+	user := &storage.User{Name: "gopher@luizalabs.com"}
+	app := &App{Name: "teresa"}
+	fake.(*FakeOperations).Storage[app.Name] = app
+
+	list, err := fake.List(user)
+	if err != nil {
+		t.Fatal("error getting app info: ", err)
+	}
+
+	if list == nil {
+		t.Fatal("expected a valid info, got nil")
+	}
+}
+
+func TestFakeOperationsListErrNotFound(t *testing.T) {
+	fake := NewFakeOperations()
+	user := &storage.User{Name: "gopher@luizalabs.com"}
+
+	if _, err := fake.List(user); teresa_errors.Get(err) != nil {
+		t.Errorf("expected ErrNotFound, got %v", teresa_errors.Get(err))
```

Comment:
I was thinking about your question @drgarcia1986 and I have one doubt. Do I really need do this test? Make sense to test if a list really exist?